atomic_cell.cc

```diff
@@ -233,6 +233,27 @@ operator<<(std::ostream& os, const atomic_cell& ac) {
     return os << atomic_cell_view(ac);
 }
 
+std::ostream&
+operator<<(std::ostream& os, const atomic_cell_view::printer& acvp) {
+    auto& type = acvp._type;
+    auto& acv = acvp._cell;
+    if (acv.is_live()) {
+        return fmt_print(os, "atomic_cell{{{};ts={:d};expiry={:d},ttl={:d}}}",
```

Comment:
This is already merged, I'll send a follow-up.

I had an idea to pretty print everything as json. This is more verbose but can easily be pretty printed and diffed when debugging something.

e.g. atomic_cell would be pretty printed as

    {'type':'atomic_cell','timestamp':4341232445,'expiry':234242,'ttl':4342}

and it would nest naturally within other json-printed objects.

Reply:
btw, this came from the non-type-aware pretty printer and is consistent with it.

I'll update both.